package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Amazon Echo skill for the Washington D.C. Metro",
   "main": "index.js",
   "scripts": {
-    "clean-deploy": "npm run clean && npm run test && npm run build && npm run deploy",
+    "clean-deploy": "npm run clean && npm run build && npm run deploy",
```

Comment:
Why did you decide to do this? I've always built something immediately after it passed tests.